NEWS.md

```diff
@@ -1,3 +1,7 @@
+# naomi 2.9.21
```

Comment:
`DESCRIPTION` has 2.9.20 but specified as 2.9.21 here. Can you confirm which? (Unsure if 2.9.20 is queued up for something else.)

Reply:
Thanks, just a typo on my end. I've fixed that in the NEWS